log/loki.go

```diff
@@ -366,7 +366,7 @@ func (h *lokiHook) createPushMessage(msgs []tmpMsg, cutOffIndex, dropped int) *l
 func (h *lokiHook) push(b bytes.Buffer) error {
 	body := b.Bytes()
 
-	req, err := http.NewRequestWithContext(context.Background(), "GET", h.addr, &b)
+	req, err := http.NewRequestWithContext(context.Background(), "POST", h.addr, &b)
```

Comment:
We should move to the http constant https://pkg.go.dev/net/http#pkg-constants
```suggestion
	req, err := http.NewRequestWithContext(context.Background(), http.MethodPost, h.addr, &b)
```